docs/source/get_started/installation.rst

```diff
@@ -57,8 +44,65 @@ using their OS package manager.
 .. _Python Downloads: http://python.org/download
 
 
+Enaml and Dependencies
+~~~~~~~~~~~~~~~~~~~~~~
+
+You can install enaml and (almost) all of its dependencies through the pip
+command line.::
+
+    $ pip install enaml
+
+Toolkit Rendering Library
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enaml's declarative widgets provide a layer of abstraction on top of the
+widgets of a toolkit rendering library. You will need to install this
+dependency separately.
+
+Enaml supports using either PyQt4 or PyQt5 and uses the `QtPy`_ library
+as compatibility layer.
+
+The recommended library is `PyQt5`_,  a robust set of Python bindings to the
+Qt 5 toolkit.  (It includes the necessary parts of Qt 5.)
+
+On 32 and 64-bit Windows, 64-bit OS X and 64-bit Linux, with Python 
+versions >=3.5, it can be installed via pip::
+
+    $ pip install pyqt5
+
+An alternative is _PyQt_, which is a robust set of Python bindings to the 
+Qt 4 toolkit. It is no longer a supported project, but can still be installed
+and used with Enaml. The `PyQt Downloads`_ page contains Windows installers
+- which also the necessary parts of Qt 4. OSX users can install PyQt4 via
+`Homebrew`_. Linux users should install via the system package manager.
+
+Another alternative is `PySide`_ bindings to Qt. This is not recommended. The
```

Comment:
Could you please distinguish between Pyside and Pyside2 ? Pyside is not longer maintained and there will be no effort to improve its support. Pyside2 is actively developed and patches to improve its support would be welcome.

Reply:
+1 PySide2 (they renamed to Qt for Python) is now officially supported by Qt. https://blog.qt.io/blog/2018/04/13/qt-for-python-is-coming-to-a-computer-near-you/  and has a more "commercial friendly" license.

Reply:
Fixed in next commit.